blades/dim_blade.h

```diff
@@ -1,13 +1,24 @@
 #ifndef BLADES_DIM_BLADE_WRAPPER_H
 #define BLADES_DIM_BLADE_WRAPPER_H
 
-class DimBladeWrapper : public BladeWrapper, BladeStyle {
+class DimBladeWrapper : public BladeWrapper, BladeStyle 
+#ifdef ENABLE_MENU_DIMBLADE
+      ,protected SaberBase
```

Comment:
I don't see why you inherit saberbase here...

Reply:
If i don't, I can not acces SaberBase::SetBrightness((float)fraction/16384.0*100.0) ??